contrib/server/operations/pbench-upload-results.py

```diff
@@ -52,32 +51,27 @@ def get_md5(tarball: Path) -> str:
 
 def upload(
     server: str, token: str, tarball: Path, metadata: Optional[list[str]] = None
-) -> Response:
-    query_parameters = {}
-
+) -> requests.Response:
     md5 = get_md5(tarball)
-    query_parameters["access"] = "public"
-    dataset = tarball.name
-    satellite, _ = tarball.parent.name.split("::")
-    meta = [
-        f"global.server.legacy.migrated:{datetime.datetime.now(tz=datetime.timezone.utc):%Y-%m-%dT%H:%M}"
-    ]
-    if satellite:
+    uploaded = datetime.datetime.fromtimestamp(
+        tarball.stat().st_mtime, tz=datetime.timezone.utc
+    )
```

Comment:
`stat()` returns the timestamp in UTC...right?  🥴

Reply:
It returns "seconds since the epoch"; and, yeah, that's UTC. However `datetime.fromtimestamp` tries to be "friendly" (I guess) and converts to *local time*.

Reply:
Well, if you omit the `tz` value, you get a "timezone naive" `datetime` value, which acts like local time, mostly, so you need that...I was just wondering whether UTC was the right value or whether we needed to do a conversion.  (I did a bunch of experimenting, and I _think_ it's all good, but my results weren't entirely conclusive.)